functions/pantheon-check-undeployed

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+# Include all of the functions that we need.
+FUNCTIONS_DIR="${BASH_SOURCE%/*}"
+if [[ ! -d "$FUNCTIONS_DIR" ]]; then FUNCTIONS_DIR="$PWD"; fi
+. "$FUNCTIONS_DIR/pantheon-script-colours"
+
+##
+# Checks that code is deployed.
+##
+check_deployed() {
+  local sitename=$1
+  local dest=$2
+  
+  echo -e "Checking for undeployed code in ${sitename}."
+  
+  #Check if previous commit was deployed
+  check=$(terminus env:code-log ${sitename}.${dest} --field=labels | sed -n -e '2{p;q}')
+
+  # Set space as the delimiter
+  IFS=', '
+
+  #Set array to compare
+  Array1=( ${dest} )
```

Comment:
Rather than using sed, create an array, and search the array, I wonder if it could be done much more simply:

1. pipe to `head` to grab the first line
2. pipe to grep to search for "live" (if the code is not on the live environment, then it's undeployed).